src/renderer/types/tabs.ts

```
@@ -3,8 +3,9 @@
  * Based on specs/001-tabbed-layout-dashboard/contracts/tab-state.ts
  */
 
-import type { Session } from './data';
 import type { TriggerColor } from '@shared/constants/triggerColors';
+import { generateUUID } from '@renderer/utils/uuid';
+import type { Session } from './data';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)